src/components/footer/footer.module.scss

```diff
@@ -0,0 +1,28 @@
+@use "breakpoints";
+
+.footer {
+  --footer-background: var(--color-engel) !important;
+  grid-column: 1 / -1;
+
+  // koros stuff
+  margin-top: -1rem;
+  > div > svg {
+    width: 100%;
```

Comment:
In the [design](https://app.abstract.com/share/2d857082-5e02-40fc-a012-46bb2a021f86?collectionId=dc6fa850-7654-4012-b593-f9b0a6c15f8a&collectionLayerId=ca264674-9b7f-4f68-a6dc-f1b6713ed0c1&mode=build&selected=root-A0912CD7-6969-494B-80D8-6221D292219D&sha=f3f5009277f1a9018a5792ff477ac0150fb1cd20) there is no koros in the footer. Adding `visibility:hidden` here will hide it.

Reply:
Ah, there seems to be thread in slack where it was stated that we will start with the default HDS footer. Comment above can be ignored.